.gitignore

```diff
@@ -58,7 +58,7 @@ coverage.xml
 
 # Django stuff:
 *.log
-
+*.sh
```

Comment:
I'm curious why you put this in the `.gitignore`. If you created a shell script that helps you, you don't want to lose it.

The repo should contain all files you want for development -- even if the deployed application doesn't need them.